ui/src/me/components/GettingStarted.scss

```diff
@@ -29,7 +29,6 @@
 }
 
 .getting-started--card {
-  position: absolute;
```

Comment:
Does removing this not break anything visually?

Reply:
i have no idea as these classes aren't scoped, was leaning on you for that 😬

Reply:
this line WAS getting overwritten on your branch in dev mode, but when production flipped the css rule order, this is what made the giant page thing